app/screens/settings/notification_settings_email/notification_settings_email.js

```diff
@@ -10,40 +10,68 @@ import {
 } from 'react-native';
 
 import {Preferences} from 'mattermost-redux/constants';
-import {getPreferencesByCategory} from 'mattermost-redux/utils/preference_utils';
+import {getEmailInterval} from 'mattermost-redux/utils/notify_props';
```

Comment:
and this file can be `app/screens/settings/notification_settings_email/notification_settings_email.ios.js`

let me know if you need help with this